ci/baremetal/baremetal-cluster.lokocfg.envsubst

```diff
@@ -44,6 +44,8 @@ cluster "bare-metal" {
   conntrack_max_per_core = 65000
 
   install_to_smallest_disk = true
+
+  download_protocol = "http"
 }
```

Comment:
As we default to https, I think we don't need it here. What do you think?

Reply:
Actually this line is ignored because it will be forced to `http` since `cached_install = "true"`